.expeditor/scripts/release_habitat/build_mac_hab_binary.sh

```diff
@@ -15,10 +15,6 @@ channel=$(get_release_channel)
 
 echo "--- Channel: $channel - bldr url: $HAB_BLDR_URL"
 
-echo "--- Installing buildkite agent"
-brew tap buildkite/buildkite
-brew install --token="$BUILDKITE_AGENT_ACCESS_TOKEN" buildkite-agent
```

Comment:
Ah, crap... sorry I forgot about this when I did https://github.com/habitat-sh/habitat/commit/e09374beebf66cb0b36da396636846c16ff908cc.

Thanks for catching it!